OpenMP.make

```diff
@@ -52,6 +52,12 @@ OMP_NEC_CPU = -fopenmp
 OMP_CRAY_NVIDIA  = -DOMP_TARGET_GPU
 OMP_CLANG_NVIDIA = -DOMP_TARGET_GPU -fopenmp=libomp -fopenmp-targets=nvptx64-nvidia-cuda
 
+INSTALLED_GPU  = $(shell /usr/lib/aomp/bin/mygpu -d gfx900)
```

Comment:
Excuse my ignorance, but what happens here if I run this on a system without AOMP installed? Does this and the below code just get skipped over?

And what about AOMP targeting any AMD ISA besides gfx900 (e.g. 906, 908)?